controllers/controllers.js

```diff
@@ -39,18 +41,32 @@ function getComments(req, res, next) {
   const { article_id } = req.params;
   fetchComments(article_id)
     .then((comments) => {
-      console.log(comments, "<-- comments");
       res.status(200).send({ comments });
     })
     .catch((err) => {
-      console.log("catch");
       next(err);
     });
 }
+
+function postComments(req, res, next) {
+  const { article_id } = req.params;
+  const { userName, body } = req.body;
+  const commentPromise = [
+    getUsers(userName),
+    fetchArticlesById(article_id),
+    printComments(body, userName, article_id),
+  ];
```

Comment:
This works, but it's actually not necessary to call the getUsers or fetchArticlesById models here. You can rely on SQL to throw an error if the username isn't in the database, whilst it will also throw an error if the article isn't found/is invalid.